### PR TITLE
HandyMate: Add service

### PR DESCRIPTION
**Pull request by yetemegn-telaye**
**In this milestone I have made the following changes:**

- [ ] Created form to add a service
- [ ] Created a method to add a service

### CODE REVIEW

File: src/store/ServicesSlice.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const addService = (service) => async (dispatch) => {
+  const response = await fetch('http://localhost:3000/api/v1/services', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(service),
+  });
+  const data = await response.json();
+  dispatch(setServices(data));
```
Comment (PrantoshB) [inline src/store/ServicesSlice.js:38]: As the `setService` action expects an array and not object, this line is creating an issue (i.e cards.map not defined). So we can replace these two lines with:

`  dispatch(fetchServices());
`


it solves the error.

**Review by PrantoshB — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ 

## 👋 Hello @yetemegn-telaye  👋


Great work on your pull request! Overall, the changes look good, but there are a few required changes that need to be addressed before merging. Please see the feedback below:

🔧 Changes Required:
Please address the feedback by making the necessary changes and pushing the updates to your branch. Once the changes are made, please mark this pull request as "Ready for Review" and tag me for another review.

💡 Tip: Remember to double-check your changes locally before pushing them. Feel free to reach out if you need any further assistance.

Thank you for your contribution! 🎉

### Cheers 🥂 
### Prantosh

**Comment by yetemegn-telaye** [comment]
**Hi @PrantoshB ,** 👋 
Thank you so much for the detailed review, it was really helpful. I have made the changes and would like you to review it again.

Yetemegn Lemma

**Review by PrantoshB — APPROVED** [review]
# Status: Approved ✅ ✅ ✅ 
Hi @yetemegn-telaye , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :octocat:  
Congratulations! 🎉 

wow

## Highlights
- Implemented the required changes ✅  

### Cheers 🥂 
### Prantosh

**Pull request merged**